homeassistant/components/subaru/__init__.py

```diff
@@ -74,11 +68,12 @@ async def async_setup_entry(hass, entry):
             ),
         )
         if not await controller.connect():
-            _LOGGER.error("Failed to connect")
-            raise ConfigEntryNotReady
-    except SubaruException as ex:
-        _LOGGER.error("Unable to communicate with Subaru API: %s", ex.message)
-        raise ConfigEntryNotReady from ex
+            raise ConfigEntryNotReady("Failed to connect")
+    except SubaruException as err:
+        raise ConfigEntryNotReady(err) from err
+
+    if DOMAIN not in hass.data:
```

Comment:
not needed after you add this again to `async_setup`

Reply:
Fixed